### DISCUSSION TITLE
Add mpd to docker image ?

### DISCUSSION THREAD

**Discussion by niawag** [jcorporation/myMPD#616] [Ideas]
Hi again, its been a long time !

I'm still using and enjoying myMPD a lot, thanks again for this great software!

I'd like to know if you could integrate mpd inside myMPD docker image, it could be easier to setup for new users and it could also guarantee that docker users are on the latest mpd version with full features.
I've found some mpd docker images and my first thought was to run mpd in another container but I tought about having both of them in the same image. Here is an example of a dockerized mpd: https://hub.docker.com/r/tobi312/rpi-mpd/

If you'd like to do this I'd be glad to help, mostly for testing but if I can do anything else let me know!

**Comment by jcorporation** [comment]
Moved to discussion.

My opinion:
- myMPD only docker image should be the standard 
- docker is designed to run only one daemon inside one container 
- providing a uptodate mpd docker image is an another project 

If you want latest myMPD + MPD you can try myMPDos. It is for aarch64 and runs perfectly on raspberry.

**Comment by niawag** [comment]
I totally understand you point and I'd be glad to test myMPDos but I'm running mpd and myMPD from a ODROID N2+ with Coreelec OS so It's not possible for me.
Do you plan on releasing a docker image of myMPDos ?

**Comment by jcorporation** [comment]
No, myMPDos would be only available as a raspberry aarch64 image. But you can find a uptodate aarch64 package for alpine in the repository to build your own docker image without compiling MPD: https://github.com/jcorporation/myMPDos/tree/master/repository/aarch64

**Comment by niawag** [comment]
Thanks for your help, I've managed to run a mpd container and make myMPD
and mpd talk to each other so I won't spend more time on it.

Le lun. 29 nov. 2021 à 22:44, Jürgen Mang ***@***.***> a
écrit :
